source/includes/kubernetes/_k8_daemonsets.md

```diff
@@ -76,3 +76,80 @@ Retrieve a daemon set and all its info in a given [environment](#administration-
 | `status`<br/>_object_                      | The status information of the daemon set                        |
 
 Note that the list is not complete, since it is refering to the [kubernetes api details](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md).
+
+<!-------------------- CREATE DAEMON SET -------------------->
+
+### Create a daemon set
+```shell
+curl -X POST \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/daemonsets"
+  Content-Type: application/json
+  {
+  "apiVersion": "apps/v1",
+  "kind": "DaemonSet",
+  "metadata": {
+    "name": "daemonset-name",
+    "namespace": "default"
+  },
+  "spec": {
+    "selector": {
+      "matchLabels": {
+        "name": "fluentd-elasticsearch"
+      }
+    },
+    "template": {
+      "metadata": {
+        "labels": {
+          "name": "fluentd-elasticsearch"
+        }
+      },
+      "spec": {
+        "containers": [
+          {
+            "name": "fluentd-elasticsearch",
+            "image": "quay.io/fluentd_elasticsearch/fluentd:v2.5.2"
+          }
+        ]
+      }
+    }
+  }
+}
+```
+
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "taskId": "1542bd45-4732-419b-87b6-4ea6ec695c2b",
+  "taskStatus": "PENDING"
+}
+```
+
+<code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/daemonsets</code>
+
+Create a daemon set in a given [environment](#administration-environments).
+
+| Required Attributes                        | &nbsp;                                                                    |
+| ------------------------------------------ | --------------------------------------------------------------------------|
+| `apiVersion` <br/> _string_                | The api version (versioned schema) of the daemon set                      |
```

Comment:
I thought `apiVersion` wasn't required?

Reply:
api version is not required by kubernetes but it IS required by cmc ;)

Reply:
its required for our API, but not for the Kubernetes API (since its in the path)